conf/TransformationZvdd-title-print.xml

```diff
@@ -52,7 +52,7 @@
 						<cgxml:literal name="medium" value="http://rdaregistry.info/termList/RDAproductionMethod/1010"/>
 						<cgxml:literal name="fulltextOnline" value="http://digi.ub.uni-heidelberg.de/diglit/falke1873"/>
 						<cgxml:literal name="title" value="Kunst und Kunstgewerbe auf der Wiener Weltausstellung 1873"/>
-						<cgxml:literal name="issued" value="1873"/>
+						<cgxml:literal name="issued" value="07.04.1873[nach dem 07.04.1873]"/>
```

Comment:
not harmonising the values will result in not so useful facets and you can't do all kinds of sorting and ranging, see e.g. https://github.com/hbz/lobid/pull/46.
I don't know what is more important, though. Best would be two have two fields: `issued` and maybe `issued-notes`.